test/unit/models/dfspTransaction.model.test.ts

```diff
@@ -447,8 +453,15 @@ describe('DFSPTransactionModel', () => {
 
     it('should throw if transactionRequestRequest is not valid', async (done) => {
       mocked(modelConfig.kvs.set).mockImplementationOnce(() => Promise.resolve(true))
-      mocked(modelConfig.dfspBackendRequests.validateThirdpartyTransactionRequest)
-        .mockImplementationOnce(() => Promise.resolve({ isValid: false }))
+      mocked(modelConfig.dfspBackendRequests.validateThirdpartyTransactionRequestAndGetContext)
+        .mockImplementationOnce(() => Promise.resolve({
```

Comment:
I believe this will need to retrieve the challenge used to register the credential with the `auth-service`
https://github.com/mojaloop/auth-service/blob/master/src/domain/consents/ID.ts#L55 

The challenge is used as an extra check atm in the `auth-service`. I believe we talked about this. 
Wheter we have that extra check or remove it is fine with me. @lewisdaly

Reply:
Hmm I have no idea why that code exists. Maybe I misunderstood you the other day, but there's no reason we should need the challenge from the credential registration step to verify a transaction.

Reply:
Ok. No problem. That if statement will need to be removed then.

Reply:
Yeah for sure.